packages/sui-js/README.md

```diff
@@ -0,0 +1,17 @@
+# sui-js
+> Set of useful js utilities
+
+```sh
+$ npm install @s-ui/js --save-dev
```

Comment:
`$ npm install @s-ui/js --save` -> it should be for production.

Reply:
right. Thanks!